main.cpp

```diff
@@ -218,7 +218,7 @@ bool llama_model_load(const std::string & fname, llama_model & model, gpt_vocab
         ctx_size += n_ctx*n_layer*n_embd*ggml_type_sizef(GGML_TYPE_F32); // memory_k
         ctx_size += n_ctx*n_layer*n_embd*ggml_type_sizef(GGML_TYPE_F32); // memory_v
 
-        ctx_size += (5 + 10*n_layer)*256; // object overhead
+        ctx_size += (5 + 10*n_layer)*hparams.n_ctx; // object overhead
```

Comment:
The object memory overhead is not related to context size. It's based on the size of `ggml_object`.

```suggestion
        ctx_size += (5 + 10*n_layer)*256; // object overhead
```